src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEEventSymbol.cs

```diff
@@ -95,7 +95,11 @@ private enum Flags : byte
 
                 const int targetSymbolCustomModifierCount = 0;
                 var typeSymbol = DynamicTypeDecoder.TransformType(originalEventType, targetSymbolCustomModifierCount, handle, moduleSymbol);
-                typeSymbol = NativeIntegerTypeDecoder.TransformType(typeSymbol, handle, moduleSymbol);
+                if (_containingType.SpecialType != SpecialType.System_Runtime_CompilerServices_RuntimeFeature)
```

Comment:
>if (_containingType.SpecialType != SpecialType.System_Runtime_CompilerServices_RuntimeFeature) [](http://example.com/codeflow?start=16&length=94)

Should this check be added to `NativeIntegerTypeDecoder.TransformType()` directly?